### PR TITLE
added a /v2 page

### PR DESCRIPTION
**Pull request by IAmMorrow**
Added a /v2 page to the dapp browser

This page implement the wallet-api-client using the react-hooks library.

This PR also implement the new EIP-1559 transaction support to the v2.

### CODE REVIEW

**Comment by adrienlacombe** [comment]
fixes https://github.com/LedgerHQ/eth-dapp-browser/issues/22

File: src/DAPPBrowserV2/Player.tsx [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+        : undefined;
+
+    // establish the selected account by order of importance
+
+    if (userRequestedAccount) {
+      return userRequestedAccount;
+    }
+
+    if (initialAccount) {
+      return userRequestedAccount;
```
Comment (IAmMorrow) [inline src/DAPPBrowserV2/Player.tsx]:
```suggestion
      return initialAccount;
```

File: manifest-local.json [hunk 2]
```diff
@@ -1,71 +1,64 @@
...
-    "domains": [
-      "https://*"
     ]
-  }
\ No newline at end of file
+  },
+  "homepageUrl": "https://market.ledger.com/",
+  "supportUrl": "https://market.ledger.com/",
+  "icon": "https://cdn.live.ledger.com/icons/platform/lmarket.png",
+  "platform": "all",
```
Comment (RamyEB) [inline manifest-local.json:31]: platforms: ["ios"..
missing visibility 
but maybe not mandatory ?

**Review by RamyEB — APPROVED** [review]
Looks good to me

File: manifest-local.json (hunk 2 above)
Comment (IAmMorrow) [reply manifest-local.json:31]: it's just a local example manifest :) Mostly used as an example

**Pull request merged**